pyomo/repn/util.py

```diff
@@ -768,16 +768,26 @@ def ordered_active_constraints(model, config):
                 row_map[id(c)] = c
         else:
             row_map[id(con)] = con
+
+    if not row_map:
+        return {}
```

Comment:
This logic is unnecessary: if `row_map` is empty, the following return statement will return an empty dict